package.json

```
@@ -1,9 +1,9 @@
 {
   "name": "vscode-language-fsh",
   "displayName": "FHIR Shorthand",
-  "description": "FHIR Shorthand (FSH) Language Support by MITRE",
+  "description": "FHIR Shorthand (FSH) Language Support by HL7 community",
```

I think we can/should just say HL7 here:
```suggestion
  "description": "FHIR Shorthand (FSH) Language Support by HL7",
```